README.md

```diff
@@ -8,26 +8,75 @@ The goal of this project is to make a feature complete Gothic game client compat
 [![Latest build](https://img.shields.io/github/release-pre/Try/opengothic?style=for-the-badge)](https://github.com/Try/opengothic/releases/latest)
 ![Screenshoot](scr0.png)
 
-### Work in progress
+#### Work in progress
 [![Build status](https://ci.appveyor.com/api/projects/status/github/Try/opengothic?svg=true)](https://ci.appveyor.com/project/Try/opengothic)
 
 The original game has been replicated fully; you can complete both the main quest and the addon.
 Check out the [bugtracker](https://github.com/Try/OpenGothic/issues) for a list of known issues.
 
-### Install on Windows
-1. Install original Gothic game from CD/Steam/GOG/etc.
-*you have to install the original game as OpenGothic does not provide any built-in game assets or game scripts*
-2. [Download latest stable build](https://github.com/Try/opengothic/releases/latest)
-3. Run '/OpenGothic/bin/Gothic2Notr.exe -g "C:\Program Files (x86)\Path\To\Gothic II"'
+Development is focused on Gothic 2 and new features are not tested for Gothic 1 compatibility. While Gothic 1 is not officially supported pull requests that fix Gothic 1 — and general — bugs are welcome.
 
-Common Gothic installation paths to be provided via `-g`:
-- "C:\Program Files (x86)\JoWooD\Gothic II"
-- "C:\Gothic II"
-- "C:\Program Files (x86)\Steam\steamapps\common\Gothic II"
-- "~/PlayOnLinux's virtual drives/Gothic2_gog/drive_c/Gothic II"
+#### Prerequisites
+An original Gothic game installation as OpenGothic does not provide any built-in game assets or scripts.
 
-### Build on Linux
-#### Dependencies
+Supported systems and graphic api's are Windows (DX12, Vulkan), Linux (Vulkan) and MacOS (Metal).
+
+#### Performance
+OpenGothic is known to run poorly on Intel integrated graphics and is barely playable (<5 Fps) on Low-End devices such as Raspberry-Pi. Mainstream graphics cards like a GTX 1060 can run the game mostly at 100+ Fps in FullHD. Perfomance related ingame menu options are `Cloud Shadows` (SSAO, off by default) and `Radial Fog` (on). Raytracing is enabled by default if capable hardware is detected. To turn it off use `-rt 0` command line argument.
```

Comment:
>It'd 40-fps on my Intel! :D

Ok I got ~20 :D. I adjust it and the options.

>to separated section.

Actually I think best would be to move this to an faq page on wiki we link to where everything which is not directly related to running/building/status of project can be looked up. Also common questions like how to limit fps etc. I could write something.

Reply:
OK, lets not dive too deep on FPS topic (also it will become obsolete quite quickly)
Let's just enlist graphics options.
Also worth mention that drawing distance is ignored :)